app/src/main/kotlin/cz/covid19cz/erouska/net/ExposureServerRepository.kt

```diff
@@ -31,7 +31,11 @@ class ExposureServerRepository(
 ) {
 
     companion object {
-        private val KEY_EXPORT_INDEX = "${AppConfig.keyExportUrl}/index.txt"
+        private val KEY_EXPORT_INDEX = if (BuildConfig.FLAVOR == "dev"){
```

Comment:
Tohle tam fakt má být? Jsem myslel že se toho ifu zbavíme.

Reply:
bohužel, kvůli nekonzistenci v BE configu